src/desktops/dskSelectMap.cpp

```diff
@@ -91,6 +109,10 @@ dskSelectMap::dskSelectMap(const CreateServerInfo& csi)
     AddTextButton(4, 590, 530, 200, 22, TC_GREEN2, _("Load game..."), NormalFont);
     // "Weiter"
     AddTextButton(5, 590, 560, 200, 22, TC_GREEN2, _("Continue"), NormalFont);
+    // random map generation
+    AddTextButton(6, 380, 530, 150, 22, TC_GREEN2, _("Random"), NormalFont);
```

Comment:
Name it more explicit "Create Random Map" or so. Even better (if you got enough time) Put "Random" on the left menu and generate like 5 random maps or so, so the user can choose and make a "Regenerate" button on the bottom that is only shown when "Random" is selected. Might be some more work and could also be done later. Just an idea...